### PR TITLE
[Enhancement] Support dy2stat for HRNet and DeeplabV3

### PR DESCRIPTION
**Pull request by 0x45f**
Support dy2stat for hrnet and deeplabV3

### CODE REVIEW

File: paddleseg/cvlibs/config.py [hunk 1]
```diff
@@ -392,6 +392,10 @@ def test_config(self) -> Dict:
     def export_config(self) -> Dict:
         return self.dic.get('export', {})
 
+    @property
+    def to_static(self) -> bool:
```
Comment (juncaipeng) [inline paddleseg/cvlibs/config.py]: 请加注释说明该参数是指是否启用动转静训练。
改为to_static_training?

Comment (0x45f) [reply paddleseg/cvlibs/config.py]: 添加了注释，修改了参数名

**Review by Aurelius84 — APPROVED** [review]
LGTM

**Review by juncaipeng — APPROVED** [review]
LGTM

**Review by wuyefeilin — APPROVED** [review]
LGTM

**Pull request merged**